### DISCUSSION TITLE
Display form field

### DISCUSSION THREAD

**Discussion by rafikApp** [Budibase/budibase#7652] [Help]
Hello everyone.
when i select an element from relationship picker it doesn't display in the embed

![image](https://user-images.githubusercontent.com/106697208/188531655-4aaa429e-a71c-4503-b413-02e6a35f759e.png)
![image](https://user-images.githubusercontent.com/106697208/188531706-6ddf1361-9e8d-43eb-8bb5-ed45ff366a28.png)

**Comment by melohagan** [comment]
Hey @rafikApp 

I think the issue may be that the binding value isn't being updated in real time. I found that the value was correctly set on a page refresh.

**Reply by rafikApp** [reply]
Hi @melohagan 
I still have the same problem.
do you have any solution please ?

**Reply by melohagan** [reply]
Maybe try removing the `<html>` and `<body>` and the doctype tags incase that's causing issues.
